components/net/fetch/methods.rs

```diff
@@ -681,12 +710,13 @@ fn handle_allowcert_request(request: &mut Request, context: &FetchContext) -> io
 
 /// [Scheme fetch](https://fetch.spec.whatwg.org#scheme-fetch)
 async fn scheme_fetch(
-    request: &mut Request,
+    fetch_params: &mut FetchParams,
     cache: &mut CorsCache,
     target: Target<'_>,
     done_chan: &mut DoneChannel,
     context: &FetchContext,
 ) -> Response {
+    let request = &mut fetch_params.request;
```

Comment:
This is step 2 in the spec, please add a spec comment.
(`// Step 2. Let request be fetchParams’s request.`)